shared/src/types/types.ts

```diff
@@ -140,7 +140,6 @@ export type ServerState =
   | "configurationNeeded"
   | "configurationReady"
   | "starting"
-  | "readyToInitialize"
   | "initializing"
```

Comment:
Should probably be dropping this one two as it appears you are removing the change to this state.

Reply:
Thanks, didn't fully know what was used where, was going to do a thorough review once basic changes are in. This helps.